test/commons/filter.test.coffee

```diff
@@ -0,0 +1,44 @@
+require("../test-suite")()
+Filter     = require('../../src/model/filter')
+Individual = require('../../src/model/individual')
+
+describe 'Filter', ->
+
+  individual = null
+  value_property = null
+
+  beforeEach ->
+    individual = {
+        type: "$INDIVIDUAL",
+        id: "ciptr4z1f00093k6krpx3vuhe",
+        attributes: {
+            name: "Unnamed",
+            type: "$INDIVIDUAL"
+        },
+        relations: { }
+      }
+
+    value_property = {
+        type: "$VALUE_PROPERTY",
+        id: "ciptr4z1g000d3k6kiybrvep4",
+        attributes:{
+          predicate: "rdfs:label",
+          object: "Unnamed",
+          type: "$VALUE_PROPERTY"
+        },
+        relations: {
+          subject: "ciptr4z1f00093k6krpx3vuhe"
+        }
+      }
+
+
+
+  it 'should init Filters', (done) ->
+    filter = new Filter()
+    done()
+
```

Comment:
For synchronous tests, you can omit the done callback, so this test would become 
```
 it 'should init Filters', ->
     filter = new Filter()
```

If the done argument is not used, chai assumes done after the last statement.